README.md

```diff
@@ -1,25 +1,19 @@
 # SmokeSignals
 CS56 Winter 2016 - Independent Android Project
 
-Original Author: [Pedro M. Sosa]
+### Project Description
+Android app that allows you to automate several tasks that can be triggered by an receiving a SMS from a different phone
 
-Authors: Franklin Tang, Jackey Lau
+### Javadoc
+file:///Users/briankim/Desktop/cs56-android-smoke-signals/javadoc/index.html
```

Comment:
javalink doc is not valid, It shows a local directory whereas it should be from github.